layouts/_default/baseof.footer.css

```diff
@@ -49,7 +49,16 @@ footer ul ul {
 footer a {
   color: white;
   text-decoration: none;
-  flex: 1 1 auto;
+  font-size: 0.875rem;
+  z-index: 1;
```

Comment:
z-index is here because some footer headers had links on us shop. Without z-index link is unclickable on mobile because toggle area was expanded to whole row.